### PR TITLE
add stack references to organizing projects and stacks guide

### PR DESCRIPTION
**Pull request by susanev**
[internal slack thread](https://pulumi.slack.com/archives/C85BS3LJZ/p1661392718349309)

### CODE REVIEW

File: themes/default/content/docs/guides/organizing-projects-stacks/_index.md [hunk 1]
```diff
@@ -70,6 +70,14 @@ different dimensions. This approach has several advantages:
   pieces that can be built independently can increase agility and improve performance, particularly when they
   evolve at different rates and/or are managed by different teams.
 
+### Using Stack References with Micro-Stacks
+
+If using the micro-stacks approach you will need a way to share information between stacks. [Stack references]({{< relref "/docs/intro/concepts/stack#stackreferences" >}}) are the Pulumi concepts you will want to use. Stack references allow you to access the outputs of one stack from another stack. Inter-Stack Dependencies allow one stack to reference the outputs of another stack.
+
+To reference values from another stack, create an instance of the StackReference type using the fully qualified name of the stack as an input, and then read exported stack outputs by their name.
+
+### Moving from a Monolithic Project Structure to Micro-Stacks
```
Comment (susanev) [inline themes/default/content/docs/guides/organizing-projects-stacks/_index.md:79]:
```suggestion
## Moving from a Monolithic Project Structure to Micro-Stacks
```

File: themes/default/content/docs/guides/organizing-projects-stacks/_index.md [hunk 2]
```diff
@@ -83,9 +91,9 @@ Here are a few, non-exhaustive, examples, of how one might go about splitting up
 
 * You may have one or more data tiers that are deployed and independently backed up.
 
-Even with this alternative breakdown, it's likely your stack structure will mirror that described earlier. For
+Even with this alternative breakdown, it's likely your stack structure will mirror a monolithic structure. For
 each project, you are apt to have multiple environments such as production, staging, testing, etc. And, indeed,
-you may have inter-dependencies between your stacks -- something that Pulumi supports in a first-class manner.
+you may have inter-dependencies between your stacks -- something that Pulumi supports in a first-class manner with stack references({{< relref "/docs/intro/concepts/stack#stackreferences" >}}).
```
Comment (susanev) [inline themes/default/content/docs/guides/organizing-projects-stacks/_index.md:96]:
```suggestion
you may have inter-dependencies between your stacks -- something that Pulumi supports in a first-class manner with [stack references]({{< relref "/docs/intro/concepts/stack#stackreferences" >}}).
```

File: themes/default/content/docs/guides/organizing-projects-stacks/_index.md [hunk 3]
```diff
@@ -70,6 +70,14 @@ different dimensions. This approach has several advantages:
...
 
+### Using Stack References with Micro-Stacks
+
+If using the micro-stacks approach you will need a way to share information between stacks. [Stack references]({{< relref "/docs/intro/concepts/stack#stackreferences" >}}) are the Pulumi concepts you will want to use. Stack references allow you to access the outputs of one stack from another stack. Inter-Stack Dependencies allow one stack to reference the outputs of another stack.
+
+To reference values from another stack, create an instance of the StackReference type using the fully qualified name of the stack as an input, and then read exported stack outputs by their name.
+
+## Moving from a Monolithic Project Structure to Micro-Stacks
+
 Here are a few, non-exhaustive, examples, of how one might go about splitting up a monolithic project structure:
```
Comment (jkodroff) [inline themes/default/content/docs/guides/organizing-projects-stacks/_index.md:81]:
```suggestion
Here are a few (non-exhaustive) examples of how one might go about splitting up a monolithic project structure:
```

File: themes/default/content/docs/guides/organizing-projects-stacks/_index.md [hunk 4]
```diff
@@ -83,9 +91,9 @@ Here are a few, non-exhaustive, examples, of how one might go about splitting up
 
 * You may have one or more data tiers that are deployed and independently backed up.
 
-Even with this alternative breakdown, it's likely your stack structure will mirror that described earlier. For
+Even with this alternative breakdown, it's likely your stack structure will mirror a monolithic structure. For
 each project, you are apt to have multiple environments such as production, staging, testing, etc. And, indeed,
-you may have inter-dependencies between your stacks -- something that Pulumi supports in a first-class manner.
+you may have inter-dependencies between your stacks -- something that Pulumi supports in a first-class manner with [stack references]({{< relref "/docs/intro/concepts/stack#stackreferences" >}}).
```
Comment (jkodroff) [inline themes/default/content/docs/guides/organizing-projects-stacks/_index.md:96]: Verified this link works when I type it out manually in my browser bar.

File: themes/default/content/docs/guides/organizing-projects-stacks/_index.md [hunk 5]
```diff
@@ -104,7 +112,7 @@ its associated Pulumi stack. Read more about
...
+Stacks have associated metadata in the form of name/value [stack tags]({{< relref "/docs/intro/concepts/stack#stack-tags" >}}). You can assign custom tags to stacks (when logged into the [Pulumi Service backend]({{< relref "/docs/intro/concepts/state" >}})) to enable grouping stacks in the [Pulumi Service](https://app.pulumi.com). For example, if you have many projects with separate stacks for production, staging, and testing environments, it may be useful to group stacks by environment instead of by project. To do this, you could assign a custom `environment` tag to each stack, assigning a value of `production` to each production stack, `staging` to each staging stack, etc. Then in the Pulumi Service, you'll be able to group stacks by `Tag: environment`.
```
Comment (jkodroff) [inline themes/default/content/docs/guides/organizing-projects-stacks/_index.md:115]: Verified the link works, same as above.

**Review by jkodroff — APPROVED** [review]

**Pull request merged**